### PR TITLE
Update orm.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Add a link to conventions for the reader.

### CODE REVIEW

File: en/orm.rst [hunk 1]
```diff
@@ -31,13 +31,17 @@ database connections <database-configuration>`.
 Quick Example
 =============
 
-To get started you don't have to write any code. If you've followed the CakePHP
-conventions for your database tables you can just start using the ORM. For
-example if we wanted to load some data from our ``articles`` table we could do::
+To get started you don't have to write any code. If you've followed the :doc:`CakePHP
+conventions for your database tables </intro/conventions.html#model-and-database-conventions>`
```
Comment (ravage84) [inline en/orm.rst]: I think we normally use references instead of URLs.

Comment (jeffblack360) [inline en/orm.rst]: Hi ravage84. I read over the RST documenation to make sure I understood which to use. I probably goofed. I'll go back and review it again.  Thanks!

Comment (jeffblack360) [inline en/orm.rst]: ravage84: I don't see a 'label' to use for the :ref: tag which is why I thought the :doc: tag was correct.  Advice?

Comment (antograssiot) [inline en/orm.rst]: http://book.cakephp.org/3.0/en/contributing/documentation.html#cross-referencing-links

doc is only working if you want to link to the top of another page I think

**Pull request merged**